README.md

```diff
@@ -37,48 +37,48 @@ This project chooses best practices as defaults:
 | Coloured output                                                                           | ✔️       | ✔️                                                           | ✔️                                                                |
 | Colourless output                                                                         | ✔️       | ✔️                                                           | ✔️                                                                |
-| `ppShow` instead of `show` for counterexample output                                      | ✔️       | ✖️                                                           | ✖️                                                                |
-| `show` for counterexample output                                                          | 🚧      | ✔️                                                           | ✔️                                                                |
-| Fancy Unicode output                                                                      | ✔️       | ✖️                                                           | ✖️                                                                |
+| `ppShow` instead of `show` for counterexample output                                      | ✔️       | ✖️                                                           | C                                                                |
+| `show` for counterexample output                                                          | 🚧      | ✔️                                                           | [Lib](https://hackage.haskell.org/package/tasty-hunit)           |
+| Fancy Unicode output                                                                      | ✔️       | ✖️                                                           | C                                                                |
 | Unicode-free output                                                                       | 🚧      | ✖️                                                           | ✔️                                                                |
-| Assertion-specific output with explanation                                                | ✔️       | ✔️                                                           | ?                                                                |
-| Coloured diffing                                                                          | ✔️       | ✔️                                                           | ✖️                                                                |
+| Assertion-specific output with explanation                                                | ✔️       | ✔️                                                           | [Lib](https://hackage.haskell.org/package/tasty-hedgehog)        |
```

Comment:
What I mean here is that when you have a `shouldSatisfy`, you should get a message that explains that a predicate should have been satisfied but wasn't